doc/nrf/releases_and_maturity/releases/release-notes-2.8.0.rst

```diff
@@ -0,0 +1,1306 @@
+  * Support for the :ref:`zephyr:nrf54l15dk_nrf54l15` board in the following samples:
+
+    * :ref:`central_bas`
+    * :ref:`bluetooth_central_hr_coded`
+    * :ref:`multiple_adv_sets`
+    * :ref:`peripheral_bms`
+    * :ref:`peripheral_cgms`
+    * :ref:`peripheral_cts_client`
+    * :ref:`peripheral_gatt_dm`
+    * :ref:`peripheral_hr_coded`
+    * :ref:`peripheral_mds`
+    * :ref:`peripheral_nfc_pairing`
+    * :ref:`power_profiling`
+    * :ref:`peripheral_rscs`
+    * :ref:`shell_bt_nus`
+    * :ref:`ble_throughput`
+    * :ref:`bluetooth_central_dfu_smp`
+    * :ref:`peripheral_status`
+
+* :ref:`bluetooth_isochronous_time_synchronization`:
+
+  * Fixed **LED** toggling issues on nRF52 and nRF53 Series devices that would occur after RTC wraps that occur every ~8.5 minutes.
+    The **LED** previously toggled unintentionally, at the wrong point in time, or not at all.
+
+* :ref:`ble_event_trigger` sample:
+
+  * Moved to the :file:`samples/bluetooth/event_trigger` folder.
+
+* :ref:`peripheral_hr_coded` sample:
+
+  * Fixed an issue where the HCI LE Set Extended Advertising Enable command was called with a NULL pointer.
+
+* :ref:`peripheral_mds` sample:
+
+  * Fixed an issue where device ID was incorrectly set during system initialization because MAC address was not available at that time.
+    The device ID is now set to ``ncs-ble-testdevice`` by default using the :kconfig:option:`CONFIG_MEMFAULT_NCS_DEVICE_ID` Kconfig option.
+
+* :ref:`ble_llpm` sample:
+
+  * Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
+
+* :ref:`bluetooth_radio_coex_1wire_sample` sample:
+
+  * Added support for the ``nrf54h20dk/nrf54h20/cpurad`` and ``nrf54l15dk/nrf54l15/cpuapp`` build targets.
+
+Bluetooth Fast Pair samples
+---------------------------
+
+* Updated:
+
+  * The values for the :kconfig:option:`CONFIG_BT_ADV_PROV_TX_POWER_CORRECTION_VAL` Kconfig option in all configurations, and for the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_TX_POWER_CORRECTION_VAL` Kconfig option in configurations with the Find My Device Network (FMDN) extension support.
+    The values are now aligned with the Fast Pair requirements.
+  * The sample configurations to use a separate workqueue for connection TX notify processing (:kconfig:option:`CONFIG_BT_CONN_TX_NOTIFY_WQ`).
+    This is done to work around the timeout in MPSL flash synchronization (``NCSDK-29354`` known issue).
+    See :ref:`known_issues` for details.
+
+* :ref:`fast_pair_locator_tag` sample:
+
+  * Added:
+
+    * LED indication on development kits for the Fast Pair advertising state.
+    * An application versioning using the :file:`VERSION` file.
+    * The DFU support which can be enabled using the ``SB_CONFIG_APP_DFU`` sysbuild Kconfig option.
+      DFU is available for all supported targets except the ``debug`` configurations of :ref:`zephyr:nrf52dk_nrf52832` and :ref:`zephyr:nrf52833dk_nrf52833` due to size constraints.
```

Comment:
Please add:
```
* An application module for the DULT motion detector feature and the new UI for supported board targets to interact with this feature.
  The development kit board targets simulate the motion with the button presses.
  The Thingy:53 target uses an 6-axis IMU with gyroscope to detect motion.
```

Reply:
This and following changes has been reviewed here: https://github.com/nrfconnect/sdk-nrf/pull/18138/